compiler/Bond/Template/CustomMapping.hs

```diff
@@ -30,11 +31,18 @@ data NamespaceMapping = NamespaceMapping
     , toNamespace :: QualifiedName
     }
 
+
```

Comment:
How about:

    type Parser a = Parsec SourceName () a

    whitespace :: Parser String
    -- etc

Then we don't need to import `Data.Monad.Identity`